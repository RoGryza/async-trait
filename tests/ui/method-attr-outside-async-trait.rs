use async_trait::async_trait;

pub trait Trait {
    #[async_trait(?Send)]
    async fn method();
}

fn main() {}
