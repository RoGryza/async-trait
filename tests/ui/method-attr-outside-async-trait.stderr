error: invalid item for #[async_trait]: expected one of: `unsafe`, `pub`, `trait`, `impl`.
       Did you forget #[async_trait] on the outer trait or impl block?
 --> tests/ui/method-attr-outside-async-trait.rs:4:5
  |
4 |     #[async_trait(?Send)]
  |     ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `async_trait` (in Nightly builds, run with -Z macro-backtrace for more info)
